src/components/PostList/index.module.scss

```diff
@@ -2,10 +2,17 @@
 
 .posts {
   width: 100%;
-  max-width: 980px;
+  max-width: 1039px;
```

Comment:
That's specific

Reply:
Yeah, the designers use a 13 columns grid but only use 11 columns. Either way, it's just a weird/random number :/